guides/source/debugging_rails_applications.md

```diff
@@ -232,23 +241,41 @@ irb(main):003:0> Article.pamplemousse
 => #<Comment id: 2, author: "1", body: "Well, actually...", article_id: 1, created_at: "2018-10-19 00:56:10", updated_at: "2018-10-19 00:56:10">
 ```
 
-Below each database statement you can see arrows pointing to the specific source filename (and line number) of the method that resulted in a database call. This can help you identify and address performance problems caused by N+1 queries: single database queries that generates multiple additional queries.
+Below each database statement you can see arrows pointing to the specific source filename (and line number) of the method that resulted in a database call e.g. `↳ app/models/article.rb:5`. This can help you identify and address performance problems caused by N+1 queries: single database queries that generates multiple additional queries.
```

Comment:
```suggestion
Below each database statement you can see arrows pointing to the specific source filename (and line number) of the method that resulted in a database call e.g. `↳ app/models/article.rb:5`.
This can help you identify and address performance problems caused by N+1 queries: single database queries that generates multiple additional queries.
```

Reply:
```diff
- This can help you identify and address performance problems caused by N+1 queries: single database queries that generates multiple additional queries.
+ This can help you identify and address performance problems caused by N+1 queries: i.e. single database queries that generate multiple additional queries.
```
(small change applied: grammar + readability.)